packages/grafana-data/src/events/pageBanner.ts

```diff
@@ -0,0 +1,27 @@
+import { BusEventWithPayload } from './types';
+
+/** @alpha */
+export class PageBannerDisplayEvent extends BusEventWithPayload<PageBannerEventPayload> {
+  static type = 'page-banner-display';
+}
+
+/** @alpha */
+export type PageBannerEventPayload = {
+  text: string;
+  closable?: boolean;
+  severity?: PageBannerSeverity;
+  trailingAction?: PageBannerAction;
```

Comment:
maybe

```suggestion
  body: React.ReactNode
```

Then the details of the banner (or list of banners!) can be handled by the supplier.

Reply:
Publishing would then look something like:

```
appEvents?.publish(
      new PageBannerDisplayEvent({
        body: <PageNotice
        text: 'Testing to display a notification in grafana',
        trailingAction: {
          text: 'Read more',
          href: 'https://www.google.se',
        } />
      })
    );
```

Reply:
Yupp, like that idea!!

Reply:
It is probably a good idea to expose the appEvents in a better way as well.